.env.dev.example

```diff
@@ -51,3 +51,6 @@ REDIS_AUTH="myredissecret"
 LANGFUSE_WORKER_PASSWORD=mybasicauthsecret
 # openssl rand -hex 32 used only here
 ENCRYPTION_KEY=0000000000000000000000000000000000000000000000000000000000000000 
+
+# speeds up local development by not executing init scripts on server startup
+NEXT_PUBLIC_LANGFUSE_RUN_NEXT_INIT_ENABLED="false"
```

Comment:
Small nitpick: Naming feels a bit redundant. "Run next init enabled" works better without the enabled. IMO we could also drop the _ENABLED here.